pageserver/src/tenant.rs

```diff
@@ -6223,8 +6226,7 @@ mod tests {
 
         let cancel = CancellationToken::new();
 
-        let mut base_key = Key::from_hex("000000000033333333444444445500000000").unwrap();
-        base_key.field1 = AUX_KEY_PREFIX;
```

Comment:
Hm, I liked that `AUX_KEY_PREFIX` was used explicitly here, it made it clear that we're testing aux files key here. Maybe construct the entire `Key` struct instead of using `from_hex`?

Reply:
I prefer put `62` in the key and then assert field1 == AUX_FILE_PREFIX to make things clear + concise.

Reply:
added an assertion that field1 == AUX_FILE_PREFIX